From b22b6d09f02f8528016935d6a78c95f565ba2865 Mon Sep 17 00:00:00 2001
From: Daniel Stonier <d.stonier@gmail.com>
Date: Tue, 12 Nov 2013 01:46:49 +0900
Subject: [PATCH] modified install rule for the deb build.

---
 debian/rules.em |    7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/debian/rules.em b/debian/rules.em
index d45dd5f..234a3b1 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -50,3 +50,10 @@ override_dh_shlibdeps:
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
 	dh_shlibdeps -l$(CURDIR)/debian/@(Package)/@(InstallationPrefix)/lib/
+
+override_dh_auto_install:
+	# In case we're installing to a non-standard location, look for a setup.sh
+	# in the install tree that was dropped by catkin, and source it.  It will
+	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
+	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
+        dh_auto_install
-- 
1.7.9.5

